packages/eslint-config-fb-strict/index.js

```diff
@@ -15,7 +15,7 @@ const variableNamePattern = String.raw`\s*[a-zA-Z_$][a-zA-Z_$\d]*\s*`;
 const importPattern = String.raw`^(?:var|let|const|import type)\s+` +
   '{?' + variableNamePattern + '(?:,' + variableNamePattern + ')*}?' +
   String.raw`\s*(?:=\s*require\(|from)[a-zA-Z_+./''\s\d\-]+\)?[^;\n]*[;\n]`;
-const maxLenIgnorePattern = String.raw`(^\s*it\(|${importPattern})`;
+const maxLenIgnorePattern = String.raw`(^\s*(it|test)\(|${importPattern})`;
```

Comment:
is this extra matching group necessary here? I'm not sure if `maxLenIgnorePattern` makes use of it